### PR TITLE
[pt-br] Update README-pt.md

### PR DESCRIPTION
**Pull request by jailton**
- Update README-pt.md following the latest english version.
- fix typos

### CODE REVIEW

File: README-pt.md [hunk 1]
```diff
@@ -153,16 +154,17 @@ make: *** [container-serve] Error 137
 
 Verifique a quantidade de memória disponível para o agente de execução de contêiner. No caso do Docker Desktop para macOS, abra o menu "Preferences..." -> "Resources..." e tente disponibilizar mais memória.
 
-# Comunidade, discussão, contribuição e apoio
+## Comunidade, discussão, contribuição e apoio
 
 Saiba mais sobre a comunidade Kubernetes SIG Docs e reuniões na [página da comunidade](http://kubernetes.io/community/).
 
-Você também pode entrar em contato com os mantenedores deste projeto em:
+Você também pode entrar em contato com os mantenedores deste projeto utlizando:
```
Comment (stormqueen1990) [inline README-pt.md:161]: Minor typo
```suggestion
Você também pode entrar em contato com os mantenedores deste projeto utilizando:
```

File: README-pt.md [hunk 2]
```diff
@@ -153,16 +154,17 @@ make: *** [container-serve] Error 137
...
 
-- [Slack](https://kubernetes.slack.com/messages/sig-docs) ([Obter o convide para o este slack](https://slack.k8s.io/))
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+    - [Obter o convide para este slack](https://slack.k8s.io/)
 - [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
 
-# Contribuindo com os documentos
+## Contribuindo com os documentos
```
Comment (stormqueen1990) [inline README-pt.md:167]: Suggestion for accuracy and readability
```suggestion
## Contribuindo com a documentação
```

File: README-pt.md [hunk 3]
```diff
@@ -179,10 +181,27 @@ Para mais informações sobre como contribuir com a documentação do Kubernetes
 * [Guia de Estilo da Documentação](http://kubernetes.io/docs/contribute/style/style-guide/)
 * [Localizando documentação do Kubernetes](https://kubernetes.io/docs/contribute/localization/)
 
-Você pode contatar os mantenedores da localização em Português em:
+### Embaixadores para novos colaboradores
+
+Caso você precise de ajuda em algum momento ao contribuir, os [Embaixadores para novos colaboradores](https://kubernetes.io/docs/contribute/advanced/#serve-as-a-new-contributor-ambassador) é uma via para contato. São aprovadores do SIG Docs cujas responsabilidades incluem orientar e ajudar novos colaboradores em seus primeiros pull requests. O melhor lugar para entrar em contato seria no [Slack do Kubernetes](https://slack.k8s.io/). Atuais Embaixadores do SIG Docs:
```
Comment (stormqueen1990) [inline README-pt.md:186]: Suggestion for readability and grammar accuracy.

```suggestion
Caso você precise de ajuda em algum momento ao contribuir, os [Embaixadores para novos colaboradores](https://kubernetes.io/docs/contribute/advanced/#serve-as-a-new-contributor-ambassador) são pontos de contato. São aprovadores do SIG Docs cujas responsabilidades incluem orientar e ajudar novos colaboradores em seus primeiros pull requests. O melhor canal para contato com embaixadores é o [Slack do Kubernetes](https://slack.k8s.io/). Atuais Embaixadores do SIG Docs:
```

File: README-pt.md [hunk 4]
```diff
@@ -179,10 +181,27 @@ Para mais informações sobre como contribuir com a documentação do Kubernetes
...
+
+## Traduções do `README.md`
+
+| Idioma                    | Idioma                     |
+| ------------------------- | -------------------------- |
+| [Alemão](README-de.md)    | [Italiano](README-it.md)   |
+| [Chinês](README-zh.md)    | [Japonês](README-ja.md)    |
+| [Coreano](README-ko.md)   | [Polonês](README-pl.md)    |
+| [Espanhol](README-es.md)  | [Português](README-pt.md)  |
+| [Françês](README-fr.md)   | [Russo](README-ru.md)      |
```
Comment (stormqueen1990) [inline README-pt.md:200]: Typo in "Francês"

```suggestion
| [Francês](README-fr.md)   | [Russo](README-ru.md)      |
```

File: README-pt.md [hunk 5]
```diff
@@ -179,10 +181,27 @@ Para mais informações sobre como contribuir com a documentação do Kubernetes
...
+| [Hindi](README-hi.md)     | [Ucraniano](README-uk.md)  |
+| [Indonésio](README-id.md) | [Vietnamita](README-vi.md) |
 
-* Felipe ([GitHub - @femrtnz](https://github.com/femrtnz))
-* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-pt)
+Você pode contatar os mantenedores da localização em Português no canal do Slack [#kubernetes-docs-pt]((https://kubernetes.slack.com/messages/kubernetes-docs-pt)).
```
Comment (stormqueen1990) [inline README-pt.md:204]: Minor typo.

```suggestion
Você pode contatar os mantenedores da localização em Português no canal do Slack [#kubernetes-docs-pt](https://kubernetes.slack.com/messages/kubernetes-docs-pt).
```

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @jailton! 👋

Thank you very much for your contribution! 🎉

I found some bits for adjustment in this page. It also seems like some parts were removed from the English version, but are still here in the Portuguese version.

I also suggest replacing all occurrences of "container" with "contêiner", as well as replacing all occurrences of "container runtime" with "agente de execução de contêiner".

If you disagree with any part of my review, please feel free to reach out on Slack. 😃

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm

File: README-pt.md [hunk 6]
```diff
@@ -2,11 +2,11 @@
...
 
-Bem-vindos! Este repositório contém todos os recursos necessários para criar o [website e documentação do Kubernetes](https://kubernetes.io/). Estamos muito satisfeitos por você querer contribuir!
+Este repositório contém todos os recursos necessários para criar o [website e documentação do Kubernetes](https://kubernetes.io/). Ficamos felizes por você querer contribuir!
 
-# Utilizando este repositório
+## Utilizando este repositório
 
-Você pode executar o website localmente utilizando o Hugo (versão Extended), ou você pode executa-ló em um container runtime. É altamente recomendável utilizar um container runtime, pois garante a consistência na implantação do website real.
+Você pode executar o website localmente utilizando o [Hugo (versão Extended)](https://gohugo.io/), ou você pode executá-lo em um agente de execução de contêiner. É altamente recomendável utilizar um agente de execução de contêiner, pois este fornece consistência de implantação em relação ao website real.
```
Comment (manoelhc) [inline README-pt.md:9]: `agente de execução` sounds a bit confusing to me and I think most of people will associate with an agent instead of runtime.

Comment (stormqueen1990) [reply README-pt.md:9]: Hi there, @manoelhc! That's the verbiage we use in the glossary for the Portuguese localization in Kubernetes docs: https://kubernetes.io/pt-br/docs/reference/glossary/?fundamental=true#term-container-runtime

**Comment by edsoncelio** [comment]
Thanks @jailton (and welcome back :D )
/approve

**Comment by seokho-son** [comment]
Hi @natalisucks 
Could you check this PR? :)
The proposed file is related with pt localization only, and the localization approvers do not have permission to approve this PR. 
(I guess l10n subgroup leads also need to have a permission to handle localization-wide issues)

**Comment by natalisucks** [comment]
Thanks for tagging me @seokho-son and for posting this in the Docs channel for a wider audience, @stormqueen1990!
/approve

**Pull request merged**